CMakeLists.txt

```diff
@@ -20,10 +23,10 @@ file(GLOB SOURCES3 "src/**/**/*.cpp")
 
 add_executable(Fun_Game ${SOURCES1} ${SOURCES2} ${SOURCES3})
 
-target_link_libraries(Fun_Game glfw3)
+target_link_libraries(Fun_Game glfw)
 target_link_libraries(Fun_Game OpenGL::GL)
 target_link_libraries(Fun_Game GLEW::GLEW)
-target_link_libraries(Fun_Game glm)
+#target_link_libraries(Fun_Game glm)
```

Comment:
Helps prevent compile errors (gives better error message for missing library), don't remove.

Reply:
glm isn't a library its a header file (I think). For some reason this breaks linux.

Reply:
The entire Standard Template *Library* is made up of headers. Header-only libraries are used with templates.

https://stackoverflow.com/questions/30055777/what-is-a-header-only-library